src/sentry/static/sentry/app/views/settings/components/forms/controls/multipleCheckbox.jsx

```diff
@@ -0,0 +1,73 @@
+import {Box} from 'grid-emotion';
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from 'react-emotion';
+
+import {defined} from '../../../../../utils';
+
+const MultipleCheckboxWrapper = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+`;
+
+const Label = styled.label`
+  font-weight: normal;
+  white-space: nowrap;
+  margin-right: 10px;
+  margin-bottom: 10px;
+  width: 20%;
+`;
+
+const CheckboxLabel = styled.span`
+  margin-left: 3px;
+`;
+
+export default class MultipleCheckbox extends React.Component {
+  static propTypes = {
+    value: PropTypes.arrayOf(PropTypes.oneOfType([PropTypes.string, PropTypes.number])),
+    onChange: PropTypes.func,
+    disabled: PropTypes.bool,
+    choices: PropTypes.array.isRequired,
+  };
+
+  onChange = (selectedValue, e) => {
+    let {value, onChange} = this.props;
+    let newValue;
+
+    if (typeof onChange !== 'function') return;
+
+    if (e.target.checked) {
```

Comment:
probably better not to rely on the dom state here?  but it's not awful

Reply:
I feel it's better to use DOM state here since this is dependent on a DOM event? thoughts?